dotcms-postman/src/main/resources/postman/Bundle_Resource.postman_collection.json

```diff
@@ -84,14 +83,423 @@
 											"});",
 											"",
 											"pm.test(\"Valid response\", function () {",
-											"    pm.expect(pm.response.text()).to.include(\"Removing bundles in a separated process\");",
+											"    pm.expect(pm.response.text()).to.include(\"The following bundles was not deleted because it does not exist: n0ex1st-18c1-4eb0-815e-981619635532\");",
```

Comment:
"The following bundles were not deleted because they don't exist: "

Reply:
done https://github.com/dotCMS/core/pull/29825/commits/914c301f07e4adce099fa0686f207eff6a282902#diff-2e4f2aadb1a2604874bcc2b2bd36e33cc4feb91991d5e7866ad2b9088e0609a4R448